src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1869,13 +1889,7 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
                 sortByCreatedDate(responseCommentList);
             }
         }
-
-        addSectionTeamNamesToTable(sectionTeamNameTable, roster, courseId, userEmail, role, feedbackSessionName, section);
-
-        return new FeedbackSessionResultsBundle(
-                session, responses, relevantQuestions, emailNameTable,
-                emailLastNameTable, emailTeamNameTable, sectionTeamNameTable,
-                visibilityTable, responseStatus, roster, responseComments, isComplete);
+        return responseComments;
     }
```

Comment:
I suspect it's the way git is displaying the diff. I'm actually extracting a block of code into a method.